resources.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+img: textbook.jpg
+img_link: http://algs4.cs.princeton.edu/home/
+caption: The course textbook
+title: resources
+active_tab: resources
+---
+
+<div>
+	<ul class="resource-header-list">
```

Comment:
I'm not sure how to fix the indentation for this one. In sublime (which I'm
using to edit), the indentation is fine.

On Mon, Aug 17, 2015 at 11:28 AM, David Xu <notifications@github.com> wrote:

> In resources.md
> <https://github.com/callison-burch/data-structures-and-algorithms/pull/1#discussion_r37220028>
> :
>
> > @@ -0,0 +1,23 @@
> > +---
> > +layout: default
> > +img: textbook.jpg
> > +img_link: http://algs4.cs.princeton.edu/home/
> > +caption: The course textbook
> > +title: resources
> > +active_tab: resources
> > +---
> > +
> > +<div>
> > +	<ul class="resource-header-list">
>
> Looks like the indentation got messed up here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/callison-burch/data-structures-and-algorithms/pull/1/files#r37220028>
> .
>